### PR TITLE
nni_enas_nlp

### PR DESCRIPTION
**Pull request by koukoulala**
Authors' implementation of "Neural Architecture Search for Text Classiﬁcation and Representation" (2019) in TensorFlow.

This project includes code for 10 text classification tasks. We conduct neural architecture search experiments in TensorFlow which redevelop the open source code of ENAS for our experiments.

This code is for running enas_nlp (Neural Architecture Search for Text Classiﬁcation and Representation) on nni.

### CODE REVIEW

File: examples/tuners/enas_nlp_nni/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ **Run ENAS_NLP in NNI**	
+ ===	
+
```
Comment (xuehui1991) [inline examples/tuners/enas_nlp_nni/README.md]: @koukoulala  Thank you for your contribution~ A little suggestion, could you provide some test metric about the 6 datasets in the README?

Comment (koukoulala) [reply examples/tuners/enas_nlp_nni/README.md]: The README has been modified. Please check again if there are any other problems.

Thank you~

Comment (xuehui1991) [reply examples/tuners/enas_nlp_nni/README.md]: @koukoulala Great!

File: examples/tuners/enas_nlp_nni/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+ **Run ENAS_NLP in NNI**	
+ ===	
+ 
+  Now we have an enas_nlp example [enas_nlp_nni](https://github.com/koukoulala/enas_nlp_nni) run in NNI from our contributors.	
+ Thanks our lovely contributors.
```
Comment (leckie-chn) [inline examples/tuners/enas_nlp_nni/README.md]: `Thanks for`

**Comment by ultmaster** [comment]
Close because there has been another plan to integrate TextNAS into NNI. Refer to #1868 and #1890.

**Pull request closed without merging**